### DISCUSSION TITLE
The port of the Admin UI must be 80,443? Can you customize the port yourself!?

### DISCUSSION THREAD

**Discussion by dsrdbh** [netbirdio/netbird#1209] [Ideas & Feature Requests]
The port of the Admin UI must be 80,443? Can you customize the port yourself!?

**Comment by mlsmaycon** [comment]
Hello @dsrdbh the ports are required for certification generation using Let's encrypt

If you are using a reverse proxy like Nginx, Traefik, or Caddy, you can avoid these ports and launch the system terminating SSL at the reverse-proxy level.
